app/src/main/res/layout/fragment_compose_dialog.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/edit_name"
```

Comment:
This name doesn't make sense to me logically and it doesn't follow the naming convention neither